rackawareness/confluent-platform.yaml

```diff
@@ -21,7 +21,7 @@ spec:
   replicas: 6
   rackAssignment:
     nodeLabels:
-    - failure-domain.beta.kubernetes.io/zone
+    - topology.kubernetes.io/zone
```

Comment:
ahh I see this:
https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesioregion

Starting in v1.17, the label `failure-domain.beta.kubernetes.io/zone` is deprecated in favor of topology.kubernetes.io/zone.

I see on my test GKE cluster that both label names are available.

If we want to support K8s versions 1.15 -> 1.20, which one label name should we specify? @xnulinu @andrewxding

Reply:
Maybe we should use the not deprecated one and make a note for k8s version less than 1.17 to use the other?

Reply:
Made note in commit  ba6a051